server/policies/collection.test.ts

```diff
@@ -64,9 +64,9 @@ describe("admin", () => {
     });
     const abilities = serialize(user, reloaded);
     expect(abilities.readDocument).toBeTruthy();
-    expect(abilities.updateDocument).toBeTruthy();
```

Comment:
@tommoor, this "bug" seems to have been on purpose cause this test case specifically tests that even in a view only collection an admin can still update the document. And the rest of the block tests other related  permissions.

Should we get rid of it anyway?

Reply:
I think we should change the behavior and tests here, admins can still give themselves edit permissions explicitly.